src/particle/__init__.py

```diff
@@ -38,3 +32,27 @@
 # Direct access to handy LaTeX to HTML particle name conversions
 # Direct access to kinematics functions
 from .particle import latex_to_html_name, lifetime_to_width, width_to_lifetime
+
+__all__ = (
+    "Geant3ID",
```

Comment:
Funny that these do not all get sorted. Almost, but not quite. No big deal.

Reply:
Only the actual imports are sorted, the `__all__` remains as-is (though I can manually sort, looks nicer that way).